### PR TITLE
i18n(fr): Update `guides/troubleshooting` 

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
 Update `guides/troubleshooting` from https://github.com/withastro/docs/commit/a2d4e24ab4868658813792d5b113142509d3f626 with typo correction.

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/troubleshooting.mdx [hunk 1]
```diff
@@ -107,7 +107,7 @@ Il se peut que l'erreur suivante apparaisse dans la console du navigateur :
 
 Cela signifie que la [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) (CSP) de votre site interdit l'exécution des balises `<script>`, qu'Astro produit par défaut.
 
-**Solution:** Mettez à jour votre CSP pour inclure `script-src : 'unsafe-inline'` afin d'autoriser l'exécution des scripts inline.
+**Solution** : Mettez à jour votre CSP pour inclure `script-src : 'unsafe-inline'` afin d'autoriser l'exécution des scripts inline. Vous pouvez également utiliser une intégration tierce telle que [`astro-shield`](https://github.com/KindSpells/astro-shield) pour générer les en-têtes CSP pour vous.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/troubleshooting.mdx:110]:
```suggestion
**Solution :** Mettez à jour votre CSP pour inclure `script-src : 'unsafe-inline'` afin d'autoriser l'exécution des scripts inline. Vous pouvez également utiliser une intégration tierce telle que [`astro-shield`](https://github.com/KindSpells/astro-shield) pour générer les en-têtes CSP pour vous.
```

**Review by dreyfus92 — COMMENTED** [review]
Merci @thomasbnt, juste une petite suggestion 😁

**Comment by thomasbnt** [comment]
> Merci @thomasbnt, juste une petite suggestion 😁

Merci @dreyfus92 !

**Review by dreyfus92 — APPROVED** [review]
Merci @thomasbnt, LGTM ✅

**Pull request merged**